scripts/translation_pipeline.py

```
@@ -362,6 +362,50 @@ def get_jobs_for_document(self, document_id: str) -> List[TranslationJob]:
         with self._lock:
             return [job for job in self.translation_jobs.values() if job.document_id == document_id]
 
+    def add_quality_report(self, job_id: str, unit_id: str, quality_score: float,
+                     issues: List[str], suggestions: List[str], reviewer: str = "system") -> str:
```

Flake8 (7.3.0)

[error] continuation line under-indented for visual indent

(E128)